bundles/org.eclipse.equinox.security.ui/src/org/eclipse/equinox/internal/security/ui/storage/TabContents.java

```diff
@@ -94,17 +95,14 @@ public TabContents(TabFolder folder, int index, final Shell shell) {
 			validateSave(); // save could fail so re-check
 		}));
 
-		/* Removed for the time being. In future modify/show/export operations could be 
-		 * re-introduced with some special access token required to be entered by the user 
-		Button buttonExport = new Button(buttonBar, SWT.CENTER);
-		buttonExport.setText(SecUIMessages.exportButton);
-		setButtonSize(buttonExport, minButtonWidth);
-		buttonExport.addSelectionListener(new SelectionAdapter() {
-			public void widgetSelected(SelectionEvent e) {
-				export();
-			}
-		});
-		*/
+		/*
+		 * Removed for the time being. In future modify/show/export operations could be
+		 * re-introduced with some special access token required to be entered by the
```

Comment:
Broken code snippet please wrap in @code .

Reply:
This is a regular java comment, not a javadoc comment, so `@code` would not make a difference here.
The only way I managed to get this preserved was using `@formatter:off` and `@formatter:on`. Is this okay?